.travis/collection.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# SPDX-License-Identifier: MIT
+
+set -e
+
+#uncomment if you use $ME - otherwise set in utils.sh
+#ME=$(basename "$0")
+SCRIPTDIR=$(readlink -f "$(dirname "$0")")
+
+. "${SCRIPTDIR}/utils.sh"
+. "${SCRIPTDIR}/config.sh"
+
+# Collection commands that are run when `tox -e collection`:
+role=$(basename "${TOPDIR}")
+toxworkdir=${1:-${TOPDIR}/.tox}
+cd "${toxworkdir}"
+toxworkdir=$(pwd)
+envlist=${2:- "yamllint,py38,shellcheck,custom"}
+
+# TEMPORARY will be replaced with the next line.
+# git clone https://github.com/linux-system-roles/auto-maintenance
+# or
+# git clone --branch some_stable_tag https://github.com/linux-system-roles/auto-maintenance
+
+# TEMPORARY
+git clone --single-branch --branch role2collection https://github.com/nhosoi/auto-maintenance
```

Comment:
I think it would be better to add ansible_role_parser.py to lsr_role2collection.py - that way we could just download a single file - we wouldn't have to use git clone we could just do something like
`curl -L -o lsr_role2collection.py https://raw.githubusercontent.com/linux-system-roles/auto-maintenance/master/lsr_role2collection.py` or `curl -L -o lsr_role2collection.py https://raw.githubusercontent.com/linux-system-roles/auto-maintenance/$STABLE_TAG/lsr_role2collection.py`

but we can worry about that later

Reply:
> I think it would be better to add ansible_role_parser.py to lsr_role2collection.py - that way we could just download a single file - we wouldn't have to use git clone we could just do something like
> `curl -L -o lsr_role2collection.py https://raw.githubusercontent.com/linux-system-roles/auto-maintenance/master/lsr_role2collection.py` or `curl -L -o lsr_role2collection.py https://raw.githubusercontent.com/linux-system-roles/auto-maintenance/$STABLE_TAG/lsr_role2collection.py`
> 
> but we can worry about that later

@richm, I've pushed another commit. Do you think it's ok to keep ansible_role_parser.py independent? Or do you think it's better be in lsr_role2collection.py?
```
Replacing git clone with curl following the suggestion by @richm.
https://github.com/linux-system-roles/template/pull/45#pullrequestreview-497822660
But ansible_role_parser.py is not added to lsr_role2collection.py.
I'm wondering ansible_role_parser.py is more useful to be a library to share?
```

Reply:
> Do you think it's ok to keep ansible_role_parser.py independent? Or do you think it's better be in lsr_role2collection.py?

I think it would be better to have 1 file, but let's get this PR merged first.

> I'm wondering ansible_role_parser.py is more useful to be a library to share?

We should have 1 file that can be imported into another python script, or run from the command line, by putting the non-reusable code in a `main()` method and changing the script like this:
```
if __name__ == "__main__":
    sys.exit(main())
```

but we can do that in a separate commit

Reply:
Thank you, @richm. Do you think it's ok to merge this PR? Or do we need some more discussions?

Reply:
> Thank you, @richm. Do you think it's ok to merge this PR? Or do we need some more discussions?

I think it's ok to merge.